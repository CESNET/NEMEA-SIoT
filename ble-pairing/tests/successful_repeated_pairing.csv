time TIMESTAMP,macaddr HCI_DEV_MAC,uint8 DATA_DIRECTION,uint8 PACKET_TYPE,bytes PACKET
2019-04-08T13:45:22.207,5c:f3:70:87:46:2b,1,4,010100
2019-04-08T13:45:27.569,5c:f3:70:87:46:2b,0,1,052006a87bc74e8f3c
2019-04-08T13:45:27.634,5c:f3:70:87:46:2b,1,4,0e0401052000
2019-04-08T13:45:27.634,5c:f3:70:87:46:2b,0,1,0b200701120012000100
2019-04-08T13:45:27.635,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:45:27.635,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:45:27.636,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:27.710,5c:f3:70:87:46:2b,1,4,3e2b0201030158a00459b32c1f1eff060001092002ae599549d38b5e62bd489ed1080058d7bbb6041c15b3daa6
2019-04-08T13:45:27.790,5c:f3:70:87:46:2b,1,4,3e2802010300bc16c7bcbd781c1bff7500420401808e78bdbcc716bcfa042ed7e93901000000000000ab
2019-04-08T13:45:28.017,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:45:28.020,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:28.020,5c:f3:70:87:46:2b,0,1,0b200700600030000001
2019-04-08T13:45:28.022,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:45:28.022,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:45:28.023,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:30.171,5c:f3:70:87:46:2b,1,4,3e2b020100013fd66be7e7d71f0201061bff570100813dea4c97b5254bce967493cc1e3a4f03d7e7e76bd63fb5
2019-04-08T13:45:30.171,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:45:30.174,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:30.174,5c:f3:70:87:46:2b,0,1,0d20196000600000013fd66be7e7d7001800280000002a0000000000
2019-04-08T13:45:30.175,5c:f3:70:87:46:2b,1,4,0f0400010d20
2019-04-08T13:45:31.461,5c:f3:70:87:46:2b,1,4,3e130100400000013fd66be7e7d7270000002a0005
2019-04-08T13:45:31.461,5c:f3:70:87:46:2b,0,1,1620024000
2019-04-08T13:45:31.465,5c:f3:70:87:46:2b,1,4,0f0400011620
2019-04-08T13:45:31.486,5c:f3:70:87:46:2b,1,2,402010000c00050012ad0800180028000000f401
2019-04-08T13:45:31.536,5c:f3:70:87:46:2b,1,4,3e0c040040000100000000000000
2019-04-08T13:45:31.536,5c:f3:70:87:46:2b,0,2,40000b00070006000104002d100d0f
2019-04-08T13:45:31.536,5c:f3:70:87:46:2b,0,2,40000a000600050013ad02000000
2019-04-08T13:45:31.536,5c:f3:70:87:46:2b,0,1,13200e4000180028000000f40100000000
2019-04-08T13:45:31.539,5c:f3:70:87:46:2b,1,4,0f0400011320
2019-04-08T13:45:31.610,5c:f3:70:87:46:2b,0,2,4000070003000400020502
2019-04-08T13:45:31.634,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:31.681,5c:f3:70:87:46:2b,1,2,4020070003000400038000
2019-04-08T13:45:31.681,5c:f3:70:87:46:2b,0,2,40000b0007000400100100ffff0028
2019-04-08T13:45:31.681,5c:f3:70:87:46:2b,1,2,40200b000700060002030001100505
2019-04-08T13:45:31.681,5c:f3:70:87:46:2b,0,2,40001500110006000357c60bc107de3e8ab8d01db1ba5a130c
2019-04-08T13:45:31.730,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:31.778,5c:f3:70:87:46:2b,1,2,4020180014000400110601000700001808000b0001180c0016000a18
2019-04-08T13:45:31.779,5c:f3:70:87:46:2b,0,2,40000b0007000400101700ffff0028
2019-04-08T13:45:31.779,5c:f3:70:87:46:2b,1,2,4020150011000600035c146ac5bce71d65f8940c0df56da02b
2019-04-08T13:45:31.779,5c:f3:70:87:46:2b,0,2,4000150011000600046d2048d7d606a141c5a4dd111307cb3d
2019-04-08T13:45:31.925,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:31.925,5c:f3:70:87:46:2b,1,2,40201a0016000400111417001c00000710af090018211235000030150000
2019-04-08T13:45:31.926,5c:f3:70:87:46:2b,0,2,40000b0007000400101d00ffff0028
2019-04-08T13:45:31.927,5c:f3:70:87:46:2b,1,4,3e0a0300400027000000f401
2019-04-08T13:45:31.974,5c:f3:70:87:46:2b,1,2,402015001100060004fc4ae61068cdce579b64f64c1b287202
2019-04-08T13:45:31.974,5c:f3:70:87:46:2b,0,1,19201c4000000000000000000000002b18eb407574e934791faba78a423d6f
2019-04-08T13:45:31.975,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:31.976,5c:f3:70:87:46:2b,1,4,0f0400011920
2019-04-08T13:45:32.022,5c:f3:70:87:46:2b,1,2,40201b002000040011061d002300111824002600021827002c000d182d0051
2019-04-08T13:45:32.023,5c:f3:70:87:46:2b,1,2,4010090000e0fe52006600e1fe
2019-04-08T13:45:32.023,5c:f3:70:87:46:2b,0,2,40000b0007000400106700ffff0028
2019-04-08T13:45:32.315,5c:f3:70:87:46:2b,1,4,080400400001
2019-04-08T13:45:32.315,5c:f3:70:87:46:2b,1,2,402015001100060006620da59420f393eabe3961a03cdf0fb6
2019-04-08T13:45:32.316,5c:f3:70:87:46:2b,1,2,40200f000b00060007d28c866169088487979e
2019-04-08T13:45:32.317,5c:f3:70:87:46:2b,1,2,40201500110006000a9ae59d2cd84b0b02f6115938f43787ce
2019-04-08T13:45:32.317,5c:f3:70:87:46:2b,0,2,400015001100060006e6f789a2b4dc5e4148de31f6e2622cfc
2019-04-08T13:45:32.317,5c:f3:70:87:46:2b,0,2,40000f000b000600071e0088ed291dfe16a111
2019-04-08T13:45:32.317,5c:f3:70:87:46:2b,0,2,40001500110006000a5e1880da87dd461a9dbc55ab8af5ab15
2019-04-08T13:45:32.364,5c:f3:70:87:46:2b,1,2,40200900050004000110ffff0a
2019-04-08T13:45:32.365,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:32.366,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:32.531,5c:f3:70:87:46:2b,0,2,4000090005000400120b000200
2019-04-08T13:45:32.607,5c:f3:70:87:46:2b,1,2,402005000100040013
2019-04-08T13:45:32.704,5c:f3:70:87:46:2b,1,4,13050140000100
2019-04-08T13:45:32.908,5c:f3:70:87:46:2b,0,2,40000700030004000a0300
2019-04-08T13:45:32.997,5c:f3:70:87:46:2b,1,2,40200e000a0004000b4d492042616e642032
2019-04-08T13:45:32.998,5c:f3:70:87:46:2b,0,2,40000700030004000a0500
2019-04-08T13:45:33.047,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:33.095,5c:f3:70:87:46:2b,1,2,40200700030004000b0000
2019-04-08T13:45:33.095,5c:f3:70:87:46:2b,0,2,40000700030004000a1600
2019-04-08T13:45:33.192,5c:f3:70:87:46:2b,1,2,40200c00080004000b01570104000001
2019-04-08T13:45:33.329,5c:f3:70:87:46:2b,1,4,13050140000100
2019-04-08T13:45:38.566,5c:f3:70:87:46:2b,0,1,052006e58f3601bb29
2019-04-08T13:45:38.631,5c:f3:70:87:46:2b,1,4,0e0401052000
2019-04-08T13:45:38.631,5c:f3:70:87:46:2b,0,1,0b200701120012000100
2019-04-08T13:45:38.632,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:45:38.632,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:45:38.633,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:38.682,5c:f3:70:87:46:2b,1,4,3e2b02010000af14da1ab1cc1f02011e1bff7500420401404eccb11ada14afceb11a6014ae01000000000000a3
2019-04-08T13:45:38.689,5c:f3:70:87:46:2b,1,4,3e1a020100011dfb9dd7ab710e02011a0aff4c0010050318bfccbfaa
2019-04-08T13:45:38.690,5c:f3:70:87:46:2b,1,4,3e0c020104011dfb9dd7ab7100aa
2019-04-08T13:45:38.694,5c:f3:70:87:46:2b,1,4,3e2b0201030158a00459b32c1f1eff060001092002ae599549d38b5e62bd489ed1080058d7bbb6041c15b3daa3
2019-04-08T13:45:38.766,5c:f3:70:87:46:2b,1,4,3e2802010300bc16c7bcbd781c1bff7500420401808e78bdbcc716bcfa042ed7e93901000000000000ad
2019-04-08T13:45:39.566,5c:f3:70:87:46:2b,0,1,060403400013
2019-04-08T13:45:39.568,5c:f3:70:87:46:2b,1,4,0f0400010604
2019-04-08T13:45:39.580,5c:f3:70:87:46:2b,1,4,050400400016
2019-04-08T13:45:39.635,5c:f3:70:87:46:2b,1,4,3e2b020100013fd66be7e7d71f0201061bff570100813dea4c97b5254bce967493cc1e3a4f03d7e7e76bd63fb7
2019-04-08T13:45:39.636,5c:f3:70:87:46:2b,1,4,3e1b020104013fd66be7e7d70f0a094d492042616e6420320302e0feb8
2019-04-08T13:45:42.881,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:45:42.949,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:42.949,5c:f3:70:87:46:2b,0,1,0b200700600030000001
2019-04-08T13:45:42.951,5c:f3:70:87:46:2b,1,4,0e04010b2000
2019-04-08T13:45:42.951,5c:f3:70:87:46:2b,0,1,0c20020101
2019-04-08T13:45:42.952,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:44.750,5c:f3:70:87:46:2b,1,4,3e2b020100013fd66be7e7d71f0201061bff570100813dea4c97b5254bce967493cc1e3a4f03d7e7e76bd63fb7
2019-04-08T13:45:44.750,5c:f3:70:87:46:2b,0,1,0c20020000
2019-04-08T13:45:44.752,5c:f3:70:87:46:2b,1,4,0e04010c2000
2019-04-08T13:45:44.752,5c:f3:70:87:46:2b,0,1,0d20196000600000013fd66be7e7d7001800280000002a0000000000
2019-04-08T13:45:44.754,5c:f3:70:87:46:2b,1,4,0f0400010d20
2019-04-08T13:45:46.033,5c:f3:70:87:46:2b,1,4,3e130100400000013fd66be7e7d7270000002a0005
2019-04-08T13:45:46.034,5c:f3:70:87:46:2b,0,1,1620024000
2019-04-08T13:45:46.037,5c:f3:70:87:46:2b,1,4,0f0400011620
2019-04-08T13:45:46.062,5c:f3:70:87:46:2b,1,2,402010000c00050012830800180028000000f401
2019-04-08T13:45:46.112,5c:f3:70:87:46:2b,1,4,3e0c040040000100000000000000
2019-04-08T13:45:46.112,5c:f3:70:87:46:2b,0,2,40000b00070006000104002d100d0f
2019-04-08T13:45:46.112,5c:f3:70:87:46:2b,0,2,40000a0006000500138302000000
2019-04-08T13:45:46.112,5c:f3:70:87:46:2b,0,1,13200e4000180028000000f40100000000
2019-04-08T13:45:46.115,5c:f3:70:87:46:2b,1,4,0f0400011320
2019-04-08T13:45:46.173,5c:f3:70:87:46:2b,0,2,4000070003000400020502
2019-04-08T13:45:46.210,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:46.257,5c:f3:70:87:46:2b,1,2,4020070003000400038000
2019-04-08T13:45:46.257,5c:f3:70:87:46:2b,0,2,40000b0007000400100100ffff0028
2019-04-08T13:45:46.258,5c:f3:70:87:46:2b,1,2,40200b000700060002030001100505
2019-04-08T13:45:46.258,5c:f3:70:87:46:2b,0,2,4000150011000600030fc9682e57c4c1310cc8dee8bce0646b
2019-04-08T13:45:46.307,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:46.355,5c:f3:70:87:46:2b,1,2,4020180014000400110601000700001808000b0001180c0016000a18
2019-04-08T13:45:46.355,5c:f3:70:87:46:2b,0,2,40000b0007000400101700ffff0028
2019-04-08T13:45:46.355,5c:f3:70:87:46:2b,1,2,402015001100060003e03da39a22650b92ea4ff09c258c32f3
2019-04-08T13:45:46.355,5c:f3:70:87:46:2b,0,2,400015001100060004f70f09e0158be3dc35fb724f3075f0c0
2019-04-08T13:45:46.404,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:46.501,5c:f3:70:87:46:2b,1,2,40201a0016000400111417001c00000710af090018211235000030150000
2019-04-08T13:45:46.501,5c:f3:70:87:46:2b,0,2,40000b0007000400101d00ffff0028
2019-04-08T13:45:46.502,5c:f3:70:87:46:2b,1,2,4020150011000600041a1625432e6da215e9bfe37351cca753
2019-04-08T13:45:46.502,5c:f3:70:87:46:2b,0,1,19201c400000000000000000000000e6b6d896dc48973c247eaa78e70478e6
2019-04-08T13:45:46.503,5c:f3:70:87:46:2b,1,4,3e0a0300400027000000f401
2019-04-08T13:45:46.504,5c:f3:70:87:46:2b,1,4,0f0400011920
2019-04-08T13:45:46.580,5c:f3:70:87:46:2b,1,4,13050140000100
2019-04-08T13:45:46.794,5c:f3:70:87:46:2b,1,4,080400400001
2019-04-08T13:45:46.794,5c:f3:70:87:46:2b,1,2,40201500110006000648fbfb3266c1496864e777fe426d85f4
2019-04-08T13:45:46.795,5c:f3:70:87:46:2b,1,2,40200f000b00060007d0ebac8fffe60a958d5c
2019-04-08T13:45:46.796,5c:f3:70:87:46:2b,1,2,40201500110006000a0053734a9e9941001c3fef167a457d8c
2019-04-08T13:45:46.796,5c:f3:70:87:46:2b,0,2,4000150011000600067b72367a7bb922342d793765899d329c
2019-04-08T13:45:46.796,5c:f3:70:87:46:2b,0,2,40000f000b00060007aa69717b565104021a67
2019-04-08T13:45:46.796,5c:f3:70:87:46:2b,0,2,40001500110006000afe1add78e97cf7104771b10b3310c131
2019-04-08T13:45:46.843,5c:f3:70:87:46:2b,1,2,40201b002000040011061d002300111824002600021827002c000d182d0051
2019-04-08T13:45:46.843,5c:f3:70:87:46:2b,1,2,4010090000e0fe52006600e1fe
2019-04-08T13:45:46.844,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:46.845,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:46.933,5c:f3:70:87:46:2b,0,2,40000b0007000400106700ffff0028
2019-04-08T13:45:46.989,5c:f3:70:87:46:2b,1,2,40200900050004000110ffff0a
2019-04-08T13:45:47.032,5c:f3:70:87:46:2b,0,2,4000090005000400120b000200
2019-04-08T13:45:47.038,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:47.086,5c:f3:70:87:46:2b,1,2,402005000100040013
2019-04-08T13:45:47.086,5c:f3:70:87:46:2b,0,2,40000700030004000a0300
2019-04-08T13:45:47.232,5c:f3:70:87:46:2b,1,2,40200e000a0004000b4d492042616e642032
2019-04-08T13:45:47.233,5c:f3:70:87:46:2b,0,2,40000700030004000a0500
2019-04-08T13:45:47.282,5c:f3:70:87:46:2b,1,4,13050140000200
2019-04-08T13:45:47.330,5c:f3:70:87:46:2b,1,2,40200700030004000b0000
2019-04-08T13:45:47.330,5c:f3:70:87:46:2b,0,2,40000700030004000a1600
2019-04-08T13:45:47.427,5c:f3:70:87:46:2b,1,2,40200c00080004000b01570104000001
2019-04-08T13:45:47.580,5c:f3:70:87:46:2b,1,4,13050140000100
