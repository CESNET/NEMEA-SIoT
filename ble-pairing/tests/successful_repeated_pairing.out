2019-04-08T13:45:32.315,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,0,0,1,0
2019-04-08T13:45:46.794,d7:e7:e7:6b:d6:3f,5c:f3:70:87:46:2b,0,1,1,0
